src/views/studio/VideoWorkspace/VideoWorkspaceForm/VideoWorkspaceForm.style.ts

```diff
@@ -9,7 +9,7 @@ import { Text } from '@/shared/components/Text'
 import { TitleArea } from '@/shared/components/TitleArea'
 import { media, sizes } from '@/shared/theme'
 
-import { EDIT_VIDEO_TABS_BAR_HEIGHT } from '../EditVideoTabsBar'
+import { EDIT_VIDEO_TABS_BAR_HEIGHT } from '../VideoWorkspaceTabsBar'
```

Comment:
Since we're renaming everything, could we change this constant to `VIDEO_WORKSPACE_TABS_BAR_HEIGHT`?